src/main/kotlin/no/nav/familie/ba/sak/kjerne/behandling/behandlingstema/BehandlingstemaService.kt

```diff
@@ -27,94 +27,79 @@ class BehandlingstemaService(
     private val vilkårsvurderingRepository: VilkårsvurderingRepository,
 ) {
     @Transactional
-    fun oppdaterBehandlingstema(
+    fun oppdaterBehandlingstemaForRegistrerSøknad(
```

Comment:
Hvis navnet skal gjenspeilet steget den brukes i så bør det kanskje byttes til

```suggestion
    fun oppdaterBehandlingstemaForRegistrereSøknad(
```